dom.bs

```diff
@@ -483,10 +483,10 @@ empty list.
 {{TouchEvent}} interface and related interfaces. [[TOUCH-EVENTS]]
 
 <p>An <a>event</a> has an associated <dfn export for=Event>path</dfn>. A <a for=Event>path</a> is a
-<a for=/>list</a> of <a for=/>structs</a>. Each <a for=/>struct</a> consists of an
-<dfn for=Event/path>item</dfn> (an {{EventTarget}} object), an
-<dfn for=Event/path>item-in-shadow-tree</dfn> (a boolean), a <dfn for=Event/path>target</dfn> (a
-<a>potential event target</a>), a
+<a for=/>list</a> of <a for=/>structs</a>. Each <a for=/>struct</a> consists of a
+<dfn for=Event/path>invocation target</dfn> (an {{EventTarget}} object), a
+<dfn for=Event/path>current-target-in-shadow-tree</dfn> (a boolean), a
```

Comment:
current target has very specific meaning, so why do we use it here too?
Shouldn't it be invocation-target-in-shadow-tree?

Reply:
You're right, thanks!